packages/labs/ssr-client/src/directives/render-light.ts

```diff
@@ -22,6 +22,7 @@ class RenderLightDirective extends Directive {
     if (typeof instance.renderLight === 'function') {
       return instance.renderLight();
     }
+    return;
```

Comment:
Do we need this? I thought TS 5.1 actually allowed you to omit returns for functions which include `undefined` in their return type? https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-1.html#easier-implicit-returns-for-undefined-returning-functions

Reply:
This started to fail with this change! I guess they loosened the check in some places and tightened it in others?

Reply:
maybe it would not be needed if `update()` method had return type annotated?